commands/commands.go

```diff
@@ -32,6 +32,32 @@ var (
 		Usage: "Indicates rolling import, which builds on and removes data to maintain a rolling 24-hour analysis",
 	}
 
+	// allFlag indicates all databases in option
+	allFlag = cli.BoolFlag{
+		Name:  "all, a",
+		Usage: "Indicates all databases should be removed",
+	}
+
+	// matchFlag indicates only matched databases to string SEARCH_STRING
+	matchFlag = cli.StringFlag{
```

Comment:
You might think about implementing these as BoolFlags so you only have to deal with reading in the search string once. No need to change it if you don't want to though.